selftests/functional/test_replay_basic.py

```diff
@@ -137,6 +181,9 @@ def test_run_replay_status_and_mux(self):
         self.assertIn(msg, result.stderr)
 
     def test_run_replay_status_and_urls(self):
+        """
+        Runs a replay job with custom urls and '--replay-test-status'.
+        """
```

Comment:
OK, this looks way cleaner than before. Still quite extensive, but manageable :smile: